templates/pages/core/get-started/intel-nuc.md

```diff
@@ -62,3 +64,27 @@ This step is only necessary in case you want to install Ubuntu Core on the emmc
  * Remove all USB flash drives and reboot the system, it will reboot from the internal memory or SSD drive now containing Ubuntu Core
 
 {% include "includes/markdown/get_started_first_boot.md" %}
+
+## Alternative install: Ubuntu Server 16.04 LTS
+
+As an alternative to Ubuntu Core, you can also install Ubuntu Server 16.04 LTS, where you can use your favourite development tools to create and run snaps.
+
+### Ubuntu image
+
+* [Intel NUC - Ubuntu Desktop 16.04 LTS image](http://people.canonical.com/~platform/snappy/nuc/ubuntu-server-16.04.1-20160817-0.iso)
+
+Download and copy the image on an USB flash drive by following the [installation media instructions](/core/get-started/installation-medias)
+
+### Installation instructions
+
+Booting the board from the USB flash drive will start the Ubuntu installer.
+
+1. Insert the USB flash drive in the NUC
+* Start the NUC and push F10 to enter the boot menu.
```

Comment:
No need, Markdown interprets it as a numbered list as long as it starts with a 1.